com.glitterware.passy.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>com.glitterware.passy</id>
```

Comment:
do the project really own `glitterware.com` ? I see no metion of it on the documentation/repository and the website at the other end is mis configured.

Reply:
No, we do not own `glitterware.com`. Can this cause problems related to publishing to Flathub? An exclusive package name can be used for Flathub, but only if necessary, since the original package name will remain for Android builds (already on F-Droid).

Reply:
Well, you said you owned it:
<img width="855" alt="image" src="https://user-images.githubusercontent.com/114441/199039094-ee89a9e6-6d0c-45fd-bb0b-79dc5454a75d.png">

It has to be changed. `io.github.glitterware.Passy` would be the acceptable one.

Reply:
e65fa69, ad30627